pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [build-system]
 requires = [
-    "setuptools>=45",
-    "setuptools_scm[toml]>=6.2",
+    "setuptools==65.5.1",
```

Comment:
The pinning is required for the fix? Or just for good measures?

Reply:
yes, it's for sanity, and I expect renovate to update them :crossed_fingers: